recipes/xenon/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "xenon" %}
+{% set version = "0.5.3" %}
+{% set sha256 = "edd793ff6a5b4af9c56899e3109285870bfdefaf8cb9cd4a6abca17d192bfbc4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
This package has entrypoints. You can add them using conda-build, https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points